app/src/main/res/anim/fade_in.xml

```diff
@@ -0,0 +1,5 @@
+<?xml version="1.0" encoding="utf-8"?>
+<alpha xmlns:android="http://schemas.android.com/apk/res/android"
+    android:fromAlpha="0.0"
+    android:toAlpha="1.0"
+    android:duration="300" />
```

Comment:
Does that reproduce the toast on your system? For me the default toast is _much_ faster than that.

Reply:
You mean that the animation of the system toast is faster? because in terms of timing I haven't noticed a real difference